healpy/src/_line_integral_convolution.pyx

```diff
@@ -0,0 +1,124 @@
+import numpy as np
+cimport numpy as np
+cimport cython
+from _common cimport RING, Healpix_Map, ndarray2map
+from pixelfunc import npix2nside
+from healpy.sphtfunc import alm2map, Alm
+
+cdef extern from "_line_integral_convolution_lib.h":
+    cdef void lic_main(const Healpix_Map[double] &Q, const Healpix_Map[double] &U,
+                       const Healpix_Map[double] &th, Healpix_Map[double] &hit,
+                       Healpix_Map[double] &tex, Healpix_Map[double] &mag,
+                       int steps, int kernel_steps, double step_radian,
+                       double polmin, double polmax)
+
+
+def line_integral_convolution(
+    Q,
+    U,
+    steps=100,
+    kernel_steps=50,
+    step_radian=0.003,
+    pol_min=-1e30,
+    pol_max=1e30,
+    rand_seed=42,
+    ell=None,
+    texture=None,
+    modulate=False,
+):
+    """Computes line integral convolution of polarized map.
+
```

Comment:
can you please explain a bit what is a line integral convolution?

Reply:
and the most important parameters like `texture`

Reply:
I added a bit more information to the doc string.